enterprise/admin.py

```diff
@@ -43,6 +54,13 @@ class EnterpriseCustomerAdmin(SimpleHistoryAdmin):
     class Meta(object):
         model = EnterpriseCustomer
 
+    @staticmethod
+    def logo(instance):
+        """
+        Instance is Enterprise customer.
+        """
+        return instance.branding_configuration.logo
```

Comment:
@asadiqbal08 This line is a bit fragile - it will only work if some external code ensures branding configuration is always created for enterprise customer. Even if such mechanism exists now, new ways of creating EC's might arise in future.

I would suggest something like 

    if instance.branding_configuration:
        return instance.branding_configuration.logo
    return None